simulation/revolve2/simulation/scene/_multi_body_system.py

```diff
@@ -135,34 +102,27 @@ def add_joint(self, joint: Joint) -> None:
 
         :param joint: The joint to add.
         """
+        maybe_index1 = self._rigid_body_to_index.get(UUIDKey(joint.rigid_body1))
```

Comment:
maybe change the `maybe_index1` to `primary_rigid_body_index` would make it more understandable i think